games-engines/openmw/openmw-9999.ebuild

```diff
@@ -38,8 +38,8 @@ RDEPEND="${LUA_DEPS}
 	media-video/ffmpeg:=
 	>=sci-physics/bullet-2.86:=[double-precision]
 	virtual/opengl
-	osg-fork? ( dev-games/openscenegraph-openmw:=[ffmpeg,jpeg,png,sdl,svg,truetype,zlib] )
-	!osg-fork? ( >=dev-games/openscenegraph-3.5.5:=[ffmpeg,jpeg,png,sdl,svg,truetype,zlib] )
+	osg-fork? ( >=dev-games/openscenegraph-openmw-3.6:=[collada(-),ffmpeg,jpeg,png,sdl,svg,truetype,zlib] )
```

Comment:
Can you fix the append-cxxflags below to use ESYSROOT and probably cppflags instead?

Reply:
Yep. For now I skipped updating 0.96 to avoid revbump and rebuild for a rare case; will remove that ebuild soon after 0.97 is released anyway